config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224764679-380dffdd-87f4-490d-ae0d-c611635be0e7.png)

- [ ] It would be nice if you add this **Acknowledgement** section in your readme file to make readme file descriptive and professional :blush: